crates/cargo-test-support/src/compare.rs

```diff
@@ -200,6 +200,27 @@ fn add_common_redactions(subs: &mut snapbox::Redactions) {
         regex!(r"ns/iter \(\+/- (?<redacted>[0-9]+(\.[0-9]+)?)\)"),
     )
     .unwrap();
+
+    // Following 3 subs redact:
+    //   "1719325877.527949100s, 61549498ns after last build at 1719325877.466399602s"
+    //   "1719503592.218193216s, 1h 1s after last build at 1719499991.982681034s"
+    // into "[DIRTY_REASON_NEW_TIME], [DIRTY_REASON_DIFF] after last build at [DIRTY_REASON_OLD_TIME]"
+    // The ordering of these 3 subs matters.
```

Comment:
The order they are inserted?  They shouldn't.  We process redactions in a priority order, rather than insertion order.

Reply:
Thanks for pointing out. Indeed, `SNAPBOX=overwrites` still works after changing order. I think I just got this misunderstanding when tweaking regex.
Fixed in 281d123